website/src/components/EventInfo.vue

```diff
@@ -52,6 +52,22 @@
             <span class="info-extended text-gray f6" v-if="link.price">{{link.price}}</span>
           </a>
         </nav>
+
+        <nav class="menu mt-4" v-if="event.resources">
```

Comment:
Because of the parsing logic, `event.resources` (and `communityResources`) is always present (but may be empty). Therefore, the boxes are always shown:

![image](https://user-images.githubusercontent.com/193136/36691146-ed631b8c-1b66-11e8-8285-ec95881b3980.png)

To fix this maybe we have to check its length as well (e.g. `v-if="event.resources && event.resources.length > 0"`), or create a method `empty: (x) => !x || !x.length` and use it in like `v-if="!empty(event.resources)"`.